pkgs/applications/misc/dotfiles/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, python27Packages }:
```

Comment:
Is explicitly python27 needed, or could it also potentially work with other versions of Python? We try to stay as generic as possible, so `pythonPackages` has preference, but if it doesn't work with 3.x, then `python2Packages` (or `python27Packages`, that doesn't really matter anymore) should be used.

Reply:
done.